clvm/more_ops.py

```diff
@@ -152,15 +152,19 @@ def op_point_add(items):
 
 
 def op_strlen(args):
+    if args.list_len() != 1:
+        raise EvalError("strlen takes exactly 1 argument", args)
     a0 = args.first()
     if a0.pair:
-        raise EvalError("len on list", a0)
+        raise EvalError("strlen on list", a0)
```

Comment:
this affects one of the existing test cases. I think `strlen` is better than `len` here.

Reply:
I agree. `strlen` is more direct.